templates/terraform/examples/route_basic.tf.erb

```diff
@@ -2,17 +2,10 @@ resource "google_compute_route" "default" {
   name        = "<%= ctx[:vars]["route_name"] %>"
   dest_range  = "15.0.0.0/24"
   network     = "${google_compute_network.default.name}"
-  next_hop_ip = "10.0.1.5"
+  next_hop_ip = "10.132.1.5"
```

Comment:
Do we have a pattern for centralizing magic variables like this that might need changing over time?

Reply:
Not that I know of!